### ISSUE TITLE
Specific combination of options causes profile pictures to poke into the sidebar

### ISSUE THREAD

**Issue by TheV360** [enchanted-sword/dashboard-unfucker#75]
**Describe the bug**
Having the **"re-add icons to reblogs"** option checked and having the **"vertical nav layout"** option unchecked causes profile pictures to poke into the sidebar. The profile pictures are interactable, which can disrupt browsing a bit.

**To Reproduce**
1. Reproduce this configuration:  
![](https://github.com/enchanted-sword/dashboard-unfucker/assets/25177071/3377dfe5-806d-4952-b795-559ed47f2499)
2. Watch the profile pictures overlap with the sidebar.

**Screenshots**
![two profile pictures intruding into the sidebar](https://github.com/enchanted-sword/dashboard-unfucker/assets/25177071/b3535997-07a2-4078-9306-0419cc9041db)

**Version**
 - Firefox 116.0.3
 - Tampermonkey 4.19.0
 - unfucker Version 3.6.3a

**Additional context**
Temporarily fixes itself if I go to another page. Going to the dashboard, the explore tab, the messages tab, and any of my blogs -- all of them will temporarily cause the column width to widen. However, refreshing causes the column width to revert back to being too narrow to fit profile pictures, though.

**Comment by enchanted-sword** [comment]
This bug was fixed in [this commit](https://github.com/enchanted-sword/dashboard-unfucker/commit/a0daac49a2767b535c5e9ec466da654b3b774287), however despite your menu showing the version as v3.6.3a it still displays the name of the toggle to disable the reblog redesign as "re-add icons to reblogs", instead of the current name, which is "revert post header changes". This seems like some sort of caching error, you might have to try manually copy/pasting the script in the script editor to update it.

**Comment by TheV360** [comment]
Aaah yeah, sorry about that -- I was on version 3.6.0 while taking those screenshots, and I noticed there was a new version, installed it, and saw that the bug (..maybe now just an invalid state) was still present in version 3.6.3. Refreshing with Ctrl+Shift+R, the bug is still present -- but if I had to guess, it's because my now-invalid preferences from the previous version are being carried over.

I guess there's another bug hidden in this, where fresh page loads don't hide incompatible options.
<details>
<summary>screenshots</summary>

**Fresh page load, with "revert vertical nav layout" saved as unchecked:**
![Screenshot from 2023-08-23 19-05-17](https://github.com/enchanted-sword/dashboard-unfucker/assets/25177071/c53fba66-0e54-4a60-a375-6dcecab950ab)
**After checking and unchecking "revert vertical nav layout":**
![Screenshot from 2023-08-23 19-05-25](https://github.com/enchanted-sword/dashboard-unfucker/assets/25177071/9a8f30d6-c86f-4c50-adea-64fc3051ec87)

</details>
